config.yml.dist

```diff
@@ -138,7 +138,9 @@ jwt:
     validation:
       iss: https://publiq-acc.eu.auth0.com/
     domain: publiq-acc.eu.auth0.com
-    
+    client_id:
+    client_secret:
```

Comment:
Same comment as before, we need appconfig and udb3-vagrant PRs for this change as well.